src/helpers/transformTokenValue.ts

```diff
@@ -3,6 +3,8 @@ import {Transaction} from 'src/interfaces/transaction';
 enum TokenID {
```

Comment:
we need to discuss this helper, i think this helper can be simplify, and make this helper dynamic token transform

Reply:
Okay, let's discuss this on discord @irmannmal